histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -106,6 +96,12 @@ def detect_tile_nuclei(slide_path, tile_position, args, it_kwargs,
 
         im_nuclei_seg_mask = htk_seg_label.delete_border(im_nuclei_seg_mask)
 
+    # Delete overlapping border nuclei
+    if any(tile_info.values()) > 0:
```

Comment:
Did you mean `if any(tile_info['tile_overlap'].values()) > 0:`?

Reply:
Yes. I've made that changes in the code but forgot to push it. That was the only addition to the code.